sui-execution/latest/sui-adapter/src/gas_meter.rs

```diff
@@ -387,6 +391,11 @@ fn enable_traverse_refs(gas_model_version: u64) -> bool {
     gas_model_version > 9
 }
 
+fn reweight_move_loc(gas_model_version: u64) -> bool {
+    // Reweighting `MoveLoc` is only done in gas model versions 10 and above.
+    gas_model_version > 10
+}
+
```

Comment:
I understand this is what we are doing, but it bothers me these aren't feature flags lol

Reply:
Same same same... I literally have a note in my backlog to come back and feature flag these things instead of basing it on gas model version...